src/components/image/Image.tsx

```diff
@@ -0,0 +1,40 @@
+import styled, { css, ThemedStyledProps } from 'styled-components/ts3.6';
+
+import { TTheme } from '../../foundations/theme-provider/themes/theme.types';
+import { applyShape } from '../../foundations/shape/Shape.mixins';
+
+import { PImage } from './Image.props';
+import ImageBase from './Image.base';
+
+const Image = styled(ImageBase)<PImage>(
+    ({
+        theme: { background, text },
+        size,
+        width,
+        height,
+        radius,
+        thumbnail,
+    }: ThemedStyledProps<PImage, TTheme>) => {
+        const thumbnailStyles = thumbnail
+            ? css`
+                  padding: 0.25rem;
+                  background-color: ${background.default};
+                  border: 1px solid ${text.disabled};
```

Comment:
I just noticed this: We should not use text values for borders. Either define a value in the theme for that (it can always be the same as `text.disabled`), or use something that is already defined for it specifically.

I would suggest the first option, since I don't think we have something like that atm.